src/webview/js/js.js

```diff
@@ -471,7 +471,7 @@ const sendScrollMessage = () => {
     startMessageId: rangeHull.first,
     endMessageId: rangeHull.last,
   });
-  if (!doNotMarkMessagesAsRead) {
+  if (shouldMarkAsReadOnScroll === 'always' || shouldMarkAsReadOnScroll === 'conversation') {
```

Comment:
This conditional is not entirely correct.

setMessagesReadAttributes(rangeHull), should only work if the chosen setting is 'always' or the settings is 'conversation' and the current narrow is a topic or pm.

Is there a way to check which narrow we are currently in the js.js file?